dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -141,7 +143,7 @@ def external_source_pipe():
             If the input has different placement (for instance, input is placed on CPU, while
             ``DALIDatasetWithInputs`` is placed on GPU) the ``tf.data.experimental.copy_to_device``
             with GPU argument must be first applied to input.
-"""
+"""  # noqa E501
```

Comment:
I'd prefer, that the lines that are too long be broken, instead of turning off the check.